subprojects/docs/src/docs/userguide/authoring-builds/writing_build_scripts.adoc

```diff
@@ -200,87 +211,119 @@ include::{snippetsPath}/tutorial/configureObjectUsingScript/tests/configureObjec
+Groovy provides shortcuts to define `List` and `Map` instances.
+For example, consider the “`apply`” method, which accepts a map parameter.
+When you apply a plugin (`apply plugin:'java'`), you don't use a map literal.
+Plugin application uses "named parameters" instead. Named parameters and map
+literals use similar syntax, but named parameters don't use wrapping brackets.
+When you call `apply`, Groovy converts the named parameter list to a map.
 
 .List and map literals
 ====
 include::sample[dir="snippets/tutorial/groovy/groovy",files="build.gradle[tags=listAndMapLiterals]"]
 ====
 
 [[sec:closures_as_the_last_parameter_in_a_method]]
-=== Closures as the last parameter in a method
+=== Last Closure Parameter Syntax
 
-The Gradle DSL uses closures in many places. You can find out more about closures https://docs.groovy-lang.org/latest/html/documentation/index.html#_closures[here]. When the last parameter of a method is a closure, you can place the closure after the method call:
+The Gradle DSL uses closures in many places.
+You can find out more about closures https://docs.groovy-lang.org/latest/html/documentation/index.html#_closures[here].
+When the last parameter of a method is a closure, you can place the closure after the method call:
 
 .Closure as method parameter
 ====
 include::sample[dir="snippets/tutorial/groovy/groovy",files="build.gradle[tags=closureAsLastParam]"]
 ====
 
 [[sec:closure_delegate]]
-=== Closure delegate
+=== Closure Delegates
 
-Each closure has a `delegate` object, which Groovy uses to look up variable and method references which are not local variables or parameters of the closure. Gradle uses this for _configuration closures_, where the `delegate` object is set to the object to be configured.
+Each closure has a `delegate` object. Groovy uses this delegate to look up variable and method
+references to nonlocal variables and closure parameters. Gradle uses this for _configuration closures_,
+where the `delegate` object refers to the object being configured.
 
 .Closure delegates
 ====
 include::sample[dir="snippets/tutorial/groovy/groovy",files="build.gradle[tags=closureDelegates]"]
 ====
 
 [[script-default-imports]]
-== Default imports
+== Default Imports
 
-To make build scripts more concise, Gradle automatically adds a set of import statements to the Gradle scripts. This means that instead of using `throw new org.gradle.api.tasks.StopExecutionException()` you can just type `throw new StopExecutionException()` instead.
+To make build scripts more concise, Gradle automatically adds a set of import statements to scripts.
+As a result, instead of writing `throw new org.gradle.api.tasks.StopExecutionException()`,
+you can write `throw new StopExecutionException()` instead.
 
-Listed below are the imports added to each script:
+The following list shows the imports Gradle adds to each script:
```

Comment:
I preferred the previous phrasing, although I can see why this might be better for some readers 👍

Reply:
```suggestion
Gradle adds the following imports to each script:
```
How about this?

Reply:
Looks good to me. If you agree, maybe we could add 'implicitly' to emphasise the point, as in 'Gradle implicitly adds ...'